dowhy/causal_refuters/dummy_outcome_refuter.py

```diff
@@ -135,6 +156,18 @@ def refute_estimate(self):
 
                 # Get the final outcome, after running through all the values in the transformation list
                 outcome_validation = self.process_data(X_train, outcome_train, X_validation, outcome_validation, self._transformation_list)
+                
+                # Check if the value of true effect has been already stored
+                # We use None as the key as we have no base category for this refutation
+                if None not in causal_effect_map:
```

Comment:
The reason, I had used ```None``` was because there was no base category, when we run the refuter on the whole dataset. Should we use an alternative like ```NO_BASE`` in place of ```None```?